core/model/src/test/java/org/eclipse/sw360/antenna/model/test/ArtifactTest.java

```diff
@@ -162,17 +161,20 @@ public void testGetFinalLicense() {
         License license2 = new License();
         license2.setName("license2");
         license2.setLongName("licenseNumber2");
+        License license3 = new License();
+        license3.setName("license3");
+        license3.setLongName("licenseNumber3");
```

Comment:
I would prefer not to change the test setup and just change the assertions. And then create a new test for the new test-case.